config/routes.rb

```diff
@@ -64,6 +64,15 @@
   get '/rails/mailers' => "rails/mailers#index"
   get '/rails/mailers/*path' => "rails/mailers#preview"
 
+  get '/software', to: 'static#software', as: :about
+  get '/faq', to: 'static#faq', as: :faq
+  get '/static/faq', to: redirect('/faq', status: 301)
```

Comment:
I redirected the former `/static/foo` URLs to `/foo`, as shorter, more readable URLs can help SEO. If there was another reason you were doing this I don't know about, just let me know and I'll revert. :smile: